README.md

```diff
@@ -1,7 +1,7 @@
 The Open Permissions Platform Query Services
 ============================================
 
-This repository contains a the query service which can be used as a reference for Open Permissions Platform Services.
+This repository contains a query service which can be used as a reference for Open Permissions Platform Services.
```

Comment:
I don't think this description is quite right - looks like it's been copied and pasted from the template service and not updated